hs/src/Reach/Eval/Core.hs

```diff
@@ -2785,12 +2786,15 @@ evalPrim p sargs =
         [] -> getBalanceOf Nothing
         [v] -> getBalanceOf . Just =<< compileCheckType T_Token v
         _ -> illegal_args
-    SLPrim_canReceive -> do
+    SLPrim_Token_accepted -> do
       at <- withAt id
       (addr_, tok_) <- two_args
+      -- TODO - The doInternalLog bit makes it so this needs to be in a consensus step.  Can I check the consensus step and generate something different for a local step?  It seems like it would be nice to be able to use this feature in both consensus and local steps.
```

Comment:
I think it is okay to not allow it locally, because it relies on consensus (network) state